cms/standard_pages/utils.py

```diff
@@ -0,0 +1,12 @@
+from django.conf import settings
+
+from cms.standard_pages.models import CookiesPage
+
+
+def get_cookies_page_url(language_code: str) -> str | None:
+    if language_code not in {supported_lang[0] for supported_lang in settings.LANGUAGES}:
```

Comment:
How about we move this above to:
```python
SUPPORTED_LANGUAGE_CODES = {lang[0] for lang in settings.LANGUAGES}
```
so that it is not recalculated every time (settings won't change during runtime)